em_workflows/config.py

```diff
@@ -62,43 +64,37 @@ class Config:
     mrc2tif_loc = os.environ.get("MRC2TIF_LOC", "/opt/rml/imod/bin/mrc2tif")
     newstack_loc = os.environ.get("NEWSTACK_LOC", "/opt/rml/imod/bin/newstack")
 
-    # environment where the app gets run - used for share selection
-    env_to_share = {
-        "dev": "RMLEMHedwigDev",
-        "qa": "RMLEMHedwigQA",
-        "prod": "RMLEMHedwigProd",
-    }
-
```

Comment:
This is of no use as of now. The `FileShareEnum` also contains the information of environment type.